js/urlsearchparams_test.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import { test, assert, assertEqual } from "./test_util.ts";
+
+test(function initString() {
```

Comment:
Can you prefix call of the test function names with “url” or “urlSearchParams”
This is so we can easily filter

Reply:
Yes, fixed in next commit 👍